bin/bundle

```diff
@@ -1,3 +1,114 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
-load Gem.bin_path('bundler', 'bundle')
+# frozen_string_literal: true
```

Comment:
the whole change doesn't look like sidekiq specific code

Reply:
It is not - but `bin/bundle` was not generated with bundler so in order to get `bin/sidekiq` generated it had to regenerate it.